build.gradle

```diff
@@ -134,6 +134,8 @@ configure(allprojects) { project ->
     testCompile "org.testcontainers:kafka:$testcontainersVersion"
     testCompile 'org.awaitility:awaitility:4.0.2'
     testCompile 'org.assertj:assertj-core:3.16.1'
+    // need this dependency because of https://issues.apache.org/jira/browse/KAFKA-10378
```

Comment:
I _think_ we should wait for 2.6.1, otherwise the users may encounter the same issue

Reply:
this is just test dependency and there s no risk for the framework because test dependencies are not transitive

Reply:
as an alternative, we could use `kafka-clients:2.5`. as far as remember it doesn't have this ssue

Reply:
I know how test dependencies work. But, if you had to add it to make the tests pass, it indicates that someone else out there may also get surprised after updating reactor-kafka and seeing runtime ClassNotFound exceptions

Reply:
ping

Reply:
@bsideup `2.6.1` is not ready yet and it looks like for `2.6.0` this is the only workaround. Should we use `kafka-clients:2.5` instead? 
I understand that we don't want to add 'temporary' dependency but we could clean up when the fix is out and for the time being, we could limit t to `testRuntimeOnly` to avoid any unattended usages.